src/model/migrations.ts

```diff
@@ -674,10 +683,43 @@ export default async function runMigrations() {
 
   migrations.push(v17);
 
+  /**
+   *************** Migration v18 ******************
+   Move favorites from local storage to react query persistent cache (AsyncStorage)
+   */
+  const v18 = async () => {
```

Comment:
very much a nit but would love this to be in the new migrations flow our king @estrattonbailey set up , allows us to defer this so it doesn't impact app start up

Reply:
Miss you guysssss ❤️

Reply:
ERIC!